Assets/Prefabs/Player.prefab

```diff
@@ -286,7 +267,7 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 8451323487123772393, guid: a93fe77e406b6bf4980621ff8accf070, type: 3}
       propertyPath: m_LocalPosition.y
-      value: 0
+      value: -0.082
```

Comment:
The player seemed to float above the ground previously, so I just moved the mesh down a bit to compensate. This may be from a navmesh thing? @thespicyboy